Storage/Bookmarks/BookmarksModel.swift

```diff
@@ -134,6 +134,7 @@ public protocol BookmarksModelFactory {
     var nullModel: BookmarksModel { get }
 
     func isBookmarked(url: String) -> Deferred<Maybe<Bool>>
+    func getURLForKeywordSearch(keyword: String) -> Deferred<Maybe<String>>
```

Comment:
@rnewman I fixed this. Though, it would help to understand why this is better. Is it because we don't care about the concept of _keyword searching_ in the `BookmarksModel` context?